spec/rails_helper.rb

```diff
@@ -75,9 +75,13 @@ class Analytics
       # rubocop:enable Style/GlobalVars
       # rubocop:disable Rails/Output
       print '                       Bundling JavaScript and stylesheets... '
-      system 'WEBPACK_PORT= yarn concurrently "yarn:build:*" > /dev/null 2>&1'
```

Comment:
Removing the environment variable isn't related to the fix, but was part of my troubleshooting. I don't think it's doing anything since the value is "empty" regardless whether it's provided or not.